docs/_book.yaml

```
@@ -13,22 +13,74 @@ upper_tabs:
     other:
     - name: Tutorials
       contents:
-      - title: Tutorials overview
-        path: /js/tutorials/
-      - title: Baz tutorial
-        path: /js/tutorials/baz_tutorial
+      - title: Getting Started
+        path: /js/tutorials/index
+      - title: Getting Setup
+        path: /js/tutorials/setup
+
+      - heading: Training Models
+      - title: Fitting a Curve to Two-Dimensional Data
+        path: /js/tutorials/training/curve
+      - title: Recognizing Handwritten Digits with CNNs
+        path: /js/tutorials/training/mnist
+      - title: Saving and Loading Models
+        path: /js/tutorials/training/mnist
+
+      - heading: Transfer Learning
+      - title: Building a webcam based game controller
+        path: /js/sample
+      - title: Building an Audio Recognizer
+        path: /js/sample
+
+      - heading: Importing Python Models
+      - title: Importing a Keras Model
+        path: /js/sample
+      - title: Importing a TensorFlow SavedModel
+        path: /js/sample
+
+      - heading: Learning Resources
+      - title: Machine Learning
+        path: /js/sample
+
     - name: Guide
       contents:
-      - title: TFJS overview
-        path: /js/overview
-      - title: Foo guide
-        path: /js/foo_guide
+      - heading: Introduction
+      - title: Overview
+        path: /js/guides/index
+      - title: FAQ
+        path: /js/sample
+
+      - heading: Fundamentals
+      - title: Tensors & Operations
+        path: /js/guides/index
+      - title: Backends
+        path: /js/sample
+
+      - heading: Models
+      - title: Models & Layers
+        path: /js/sample
+      - title: Training Models
```

```suggestion
      - title: Train models
```

Training is also ok here

---

Dine